idunn/utils/categories.yml

```diff
@@ -0,0 +1,30 @@
+categories:
+    restaurant:
+        raw_filters:
+            - "*,restaurant"
+            - "*,cafe"
+            - "*,fast_food"
```

Comment:
Maybe we should rely on classes, instead of subclasses when possible. Both for retro-compatibility and maintainability.  
`restaurant,*`, `cafe,*`, etc